src/gt4py/storage/storage.py

```diff
@@ -39,659 +36,105 @@ def _error_on_invalid_backend(backend):
         raise RuntimeError(f"Backend '{backend}' is not registered.")
 
 
-def empty(backend, default_origin, shape, dtype, mask=None, *, managed_memory=False):
+def empty(backend, aligned_index, shape, dtype, dimensions=None):
```

Comment:
Can `aligned_index` be made optional and default to `(0, 0, 0)`?

Reply:
Docstrings done.

Reply:
Sorry to bother, but I still don't see docstrings on `empty`, `ones`, and `zeros` here. Can you add those before merging? Thanks!

Reply:
They're there! I think if you click the file name above it takes you to the old version.